dgraph/cmd/mcp/mcp_server.go

```diff
@@ -70,6 +72,64 @@ func NewMCPServer(connectionString string, readOnly bool) (*server.MCPServer, er
 		}),
 	)
 
+	queryCheckTool := mcp.NewTool("check_query",
```

Comment:
Maybe this should be more accurately named `check_query_syntax` or even better IMO `validate_query_syntax`